website/src/css/custom.css

```diff
@@ -62,3 +62,14 @@
 .anchor {
   scroll-margin-top: 50pt;
 }
+
+@media (max-width: 996px) {
+  .navbarSearchContainer_node_modules-\@docusaurus-theme-classic-lib-theme-Navbar-Search-styles-module  {
+    position: relative !important;
+    right: 0 !important;
```

Comment:
have to use `!important` since the CSS for this element are lower down the CSS cascade and can't be overwritten otherwise